app/models/user_journey.rb

```diff
@@ -29,6 +29,10 @@ def all_steps_completed?
     journey.steps.count == user_steps.completed.count
   end
 
+  def started?
+    !(user_steps.count == 1 && user_steps.first.not_started?)
```

Comment:
Tato logika podla mna nie je uplne dobra. Nestaci skontrolovat toto? https://github.com/slovensko-digital/navody.digital/blob/master/app/models/user_journey.rb#L8

Reply:
Toto som dorábala na základe testov. Nastala situácia, kedy používateľ označil prvý task ako done, potom undone. V takomto prípade je k task-u jeden záznam v db, ktorý má status "not_started". Ale ako nižšie píše @michal-rohacek , má to ešte nedostatky. Fixnem.

Reply:
Rozumiem, ale ked to niekto spravi, tak by som to asi povazoval za zacatu user journey aj tak. Nekomplikujme si zivot, ked na to raz niekto klikne tak to povazujme za zacate.

Reply:
Medzitým som to opravila podľa návrhu nižšie, takže je skontrolované či náhodou všetky kroky nie sú 'not_started'. Ak treba, môžem to vyhodiť a budeme takú úlohu považovať za začatú.

Reply:
Som za to, ze to zjednodusme. V prehlade vlastne potrbeujeme len odfiltrovat tie user_journeys, ktore su uz cele hotove.

Reply:
Zmazané.